packages/hardhat-plugin/test/ui/helpers/calculate-deploying-module-panel-display.ts

```diff
@@ -21,6 +21,9 @@ describe("ui - calculate starting message display", () => {
     maxFeeBumps: 0,
     gasBumps: {},
     strategy: "basic",
+    ledger: false,
+    ledgerMessage: "",
+    ledgerMessageIsDisplayed: false,
```

Comment:
Is the `ledger` property needed in addition to `ledgerMessageIsDisplayed`?
What drives the difference between the `ledger: true` state and the `ledgerMessageIsDisplayed: false` state?